eda_plugin/utility/eda_gui.py

```diff
@@ -22,7 +22,8 @@
 log = logging.getLogger("EDA")
 
 # Adjust for different screen sizes
-QtWidgets.QApplication.setAttribute(QtCore.Qt.HighDpiScaleFactorRoundingPolicy.PassThrough)
+QtWidgets.QApplication.setHighDpiScaleFactorRoundingPolicy(QtCore.Qt.HighDpiScaleFactorRoundingPolicy.PassThrough)
```

Comment:
note:
this is the one thing I couldn't go back and check on pyqt5... but it does look like this is the API that was there in both 5 and 6: https://doc.qt.io/qt-5/qguiapplication.html#setHighDpiScaleFactorRoundingPolicy

Reply:
I think I saw recently that this setting is now standard in PyQt6